scripts/package.json

```diff
@@ -4,7 +4,7 @@
   "version": "0.0.0",
   "license": "UNLICENSED",
   "dependencies": {
-    "@adeira/js": "^1.2.0",
+    "@adeira/js": "^1.2.4",
```

Comment:
might be worth adding some tests for this -> `yarn workspaces info` => dependency is added to `mismatchedWorkspaceDependencies` in such cases

Reply:
Feel free to create a PR for it 🙂